src/arguments.rs

```diff
@@ -19,7 +20,12 @@ pub fn run() -> Result<()> {
             SubCommand::Rename { old_name, new_name } => commands::rename(&old_name, &new_name)?,
         }
     } else {
-        commands::current()?;
+        if fzf::is_fzf_installed() {
+            let choice = fzf::fuzzy_find_config().unwrap();
+            commands::activate(&choice)?
```

Comment:
This means that the "argumentless" command differs depending on whether `fzf` is installed or not. If not it works the same as `current` but if so it works like `activate`. I'd rather keep that consistent to mean `current`.

Perhaps in future we could have a config value for setting the default command so that it can work either way, but for now I prefer the default remaining `current`.

I'd prefer `gctx activate` with no arguments to launch `fzf` to prompt for a name if `fzf` is installed, and give an error if not.

Reply:
Think this is resolved in the last commit.